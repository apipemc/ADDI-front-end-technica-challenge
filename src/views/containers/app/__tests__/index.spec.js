import React from 'react';
import { shallow } from 'enzyme';

import AppContainer from '../index';

describe('AppContainer', () => {
  it('should render a view', () => {
    const wrapper = shallow(
      <AppContainer>
        <span>Children</span>
      </AppContainer>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
