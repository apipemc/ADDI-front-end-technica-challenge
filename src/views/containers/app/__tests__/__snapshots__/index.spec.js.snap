// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer should render a view 1`] = `
<div
  className="container"
>
  <Header />
  <main
    className="content__main"
  >
    <span>
      Children
    </span>
  </main>
  <ToastContainer
    autoClose={5000}
    bodyClassName={null}
    className={null}
    closeButton={
      <CloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="top-right"
    progressClassName={null}
    progressStyle={null}
    role="alert"
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
</div>
`;
